corehq/sql_db/routers.py

```diff
@@ -102,9 +107,12 @@ def db_for_read_write(model, write=True):
     elif app_label == SYNCLOGS_APP:
         return settings.SYNCLOGS_SQL_DB_ALIAS
     elif app_label == ICDS_REPORTS_APP:
-        engine_id = ICDS_UCR_ENGINE_ID
-        if not write:
-            engine_id = connection_manager.get_load_balanced_read_db_alias(ICDS_UCR_ENGINE_ID)
+        if getattr(_thread_local, 'force_citus', False):
+            engine_id = CITUS_MASTER
+        else:
+            engine_id = ICDS_UCR_ENGINE_ID
+            if not write:
+                engine_id = connection_manager.get_load_balanced_read_db_alias(ICDS_UCR_ENGINE_ID)
         return connection_manager.get_django_db_alias(engine_id)
```

Comment:
This wat, doesn't the other uninteded ICDS models also get routed to Citus, like `ICDSAuditEntryRecord` etc for which we are not doing citus writes?